otwcode/otwarchive
# AO3-4913 Increase coverage of ChallengeSignupsController

## Issue

https://otwarchive.atlassian.net/browse/AO3-4913

## Purpose

Increase coverage of ChallengeSignupsController

## Testing

No manual testing required.

challenge_claims_controller currently has 70.11% test coverage on Coveralls:
https://coveralls.io/builds/10550066/source?filename=app%2Fcontrollers%2Fchallenge_signups_controller.rb
We'd like to increase that, with an ultimate goal of 90% or greater.
How to test:
Use Coveralls to confirm that the coverage percentage has increased:
1. Go to https://coveralls.io/github/otwcode/otwarchive?branch=master 
2. In the "Search" field near the "Source Files on Master" heading, enter the name of the controller: challenge_claims_controller
3. Note the number in the "Coverage" column of the table
